src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+
+import {ArrowLeft, ArrowRight, Xmark} from '@gravity-ui/icons';
+import type {ModalProps} from '@gravity-ui/uikit';
+import {Button, Icon, Modal, Text, ThemeProvider, useThemeValue} from '@gravity-ui/uikit';
+
+import {block} from '../utils/cn';
+
+import type {GalleryItemProps} from './GalleryItem';
+import {FilesGalleryFallbackText} from './components/FallbackText';
+import type {UseNavigationProps} from './hooks/useNavigation';
+import {useNavigation} from './hooks/useNavigation';
+import {i18n} from './i18n';
+import {getInvertedTheme} from './utils/getInvertedTheme';
+
+import './Gallery.scss';
+
+const cnGallery = block('gallery');
+
+const emptyItems: GalleryItemProps[] = [];
+
+export type GalleryProps = {
+    fullScreen?: boolean;
+    modalClassName?: string;
+    className?: string;
+    children?: React.ReactElement<GalleryItemProps>[];
+    invertTheme?: boolean;
```

Comment:
There is no need to add it into the component, the user can easily do it by himself:
```
<ThemeProvider>
  <Gallery/>
</ThemeProvider>
```